editor/extensions/ext-overview_window.js

```diff
@@ -105,7 +111,7 @@ svgEditor.addExtension("overview_window", function() {	'use strict';
 		,stop :function(){overviewWindowGlobals.viewBoxDragging=false;}
 	});  
 	$("#overviewMiniView").click(function(evt){
-		//Firefox doesn't support evt.offsetX and evt.offsetY
+		// Firefox doesn't support evt.offsetX and evt.offsetY.
```

Comment:
This pull request is fine, but this comment is now stale. Firefox supports event.offsetX as of July this year.